src/charm.py

```diff
@@ -118,6 +133,49 @@ def __init__(self, *args):
             self._certificates.on.certificate_expiring, self._on_certificate_expiring
         )
 
+    def _configure_amf(self, _ : EventBase) -> None:
+        """Handle Juju events.
+
+        This event handler is called for every event that affects the charm state
+        (ex. configuration files, relation data). This method performs a couple of checks
+        to make sure that the workload is ready to be started. Then, it configures the AMF
+        workload, runs the Pebble services and expose the service information through
+        charm's interface.
+
+        Args:
+            _ (EventBase): Juju event
```

Comment:
```suggestion
```
Do we need this part?

Reply:
I think we need  to keep the Args part as this method takes argument.